clients/runner/src/runner.rs

```diff
@@ -515,10 +513,6 @@ impl RunnerExt for Runner {
 		&self.opts.download_path
 	}
 
-	fn parachain_url(&self) -> String {
```

Comment:
This was found to be unused. I also checked, and found it unused anywhere in the code. Please confirm @gianfra-t

Reply:
We remove some logic from the original runner code, so it makes sense that some methods are not used. I think it is fine!